documentation/extensions/dataforms.md

```diff
@@ -106,22 +112,30 @@ the message to send back.
 **Examples**
 
 In this example we can see how to retrieve a form to fill out, complete the
-form and send it back:
+form, and send it back:
```

Comment:
is this comma necessary?

Reply:
I thought the documentation was describing three separate items in succession.  I am no English expert by any stretch but from what I recall I thought we needed the commas to separate items in the list.  I can remove it, if you would like.

Reply:
I'm not a native speaker, so I can only deduce from German, where the comma is unnecessary :D
I'm fine with your change though :)